### PR TITLE
added stairs exam 2018-01-17 with FRR

### PR DESCRIPTION
**Pull request by Francesco9932**

### CODE REVIEW

File: Exam Labs/2018-01-17 Stairs/lab/as1r1/etc/frr/frr.conf [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+!
+!  BGP CONFIGURATION
+!
+debug bgp keepalives
+debug bgp updates in
+debug bgp updates out
+!
+router bgp 1
+no bgp ebgp-requires-policy
+no bgp network import-check
```
Comment (lorenzo93) [inline Exam Labs/2018-01-17 Stairs/lab/as1r1/etc/frr/frr.conf:20]: Not needed, the announced prefixes are the same as the interfaces prefixes

File: Exam Labs/2018-01-17 Stairs/lab/as1r2/etc/frr/frr.conf [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+!
+! NETWORKS
+!
+network 1.0.0.0/24
+network 10.0.0.4/30
+
+neighbor 10.0.0.6 route-map metricOut out
+
+route-map metricOut permit 10
+set metric 10
```
Comment (lorenzo93) [inline Exam Labs/2018-01-17 Stairs/lab/as1r2/etc/frr/frr.conf:37]: Please, add indentation to improve readability. (you can find the correct indentation running the command show running-config inside vtysh)

File: Exam Labs/2018-01-17 Stairs/lab/lab.conf [hunk 3]
```diff
@@ -0,0 +1,28 @@
+as1r1[0]="A"
```
Comment (lorenzo93) [inline Exam Labs/2018-01-17 Stairs/lab/lab.conf:1]: Please, add at the start of the lab.conf the metadata (you can see an example other labs) and feel free to put your name as author :)

**Review by lorenzo93 — CHANGES_REQUESTED** [review]
Hi @Francesco9932 

thanks very much for your work, we appreciate it a lot.
I've made some comments on your code. Notice that, some of them are just on one file but they should be seen as "global" comments.

**Comment by Francesco9932** [comment]
Hi @lorenzo93 

Thanks for your appreciation of my and George work in making this lab. 
I fixed everything you suggested me and i pushed the changes.

**Comment by lorenzo93** [comment]
Hi @Francesco9932 ,

I've checked your lab and everything works fine.
Just two small changes and I'll merge your PR.

1. Remember to update the `lab.zip` file with the new changes
2. Please, to improve readability, add indentation to the `frr.conf`files. (You can find the correct indentation running the command `show running-config` inside `vtysh`)

**Comment by Francesco9932** [comment]
Hi @lorenzo93 i've added the correct indentation in frr.conf and i've updated the lab.zip version of the lab with the new changes that you suggested me.

**Review by lorenzo93 — APPROVED** [review]

**Pull request merged**